### DISCUSSION TITLE
[feature] Export data (as csv) for all charts in a dashboard

### DISCUSSION THREAD

**Discussion by sebastianliebscher** [apache/superset#25370] [Ideas]
> In apache superset, I see an 'Export CSV' option to export data as CSV for an individual chart. I have a dashboard which has many charts and I want to allow users to export data from all the charts in a single click. What would the easiest approach to achieve the same?
> 
> I see the /superset/explore_json?csv=true API can be invoked to export data for a chart. Can it be used for doing same for multiple charts in a single invocation? Or any other better approach?

This is a [question on StackOverflow](https://stackoverflow.com/q/63632220/22257868) with about 4k views. I think it is worth mentioning here.

**Comment by r3nt3r** [comment]
Almost a year and no one found it useful?

**Comment by mdsharpe** [comment]
Sounds pretty useful to me 👍
